commands/command_migrate_export.go

```diff
@@ -45,15 +46,26 @@ func migrateExportCommand(cmd *cobra.Command, args []string) {
 				return b, nil
 			}
 
-			var buf bytes.Buffer
+			ptr, err := lfs.DecodePointer(b.Contents)
+			if errors.IsNotAPointerError(err) {
```

Comment:
Nit; let's push this down into the below conditional, since an error will never be `nil` and pass `errors.IsNotAPointerError(...)`.